src/main/frontend/components/block.cljs

```diff
@@ -1334,6 +1334,8 @@
 
 (defn- bullet-on-click
   [e block uuid]
+  (if (-> block :block/properties :collapsed)
+    (editor-handler/expand-all! uuid (:block/level block)))
```

Comment:
This place can use `editor-handler/expand-block!` directly.
But sometimes the code to expand block doesn't work

Reply:
editor-handler/expand-block! is a private function.